src/pages/settings/PasswordPage.js

```diff
@@ -180,7 +181,7 @@ class PasswordPage extends Component {
                                             value={this.state.currentPassword}
                                             onChangeText={text => this.clearErrorAndSetValue('currentPassword', text)}
                                             returnKeyType="done"
-                                            hasError={this.state.errors.currentPassword}
+                                            hasError={this.state.errors.currentPassword || this.props.account.errorInCurrentPassword}
                                             errorText={this.getErrorText('currentPassword')}
```

Comment:
`errorText` should be removed if we're following the new styling conventioned outlined in [this comment](https://github.com/Expensify/App/issues/8440#issuecomment-1103186938)

Otherwise we still get the following display 

<img width="366" alt="image" src="https://user-images.githubusercontent.com/16747822/168693110-f72e11d1-ec9c-444c-8ca0-58a4101719ec.png">

Reply:
I feel like this is fine as it is only showing the required field error.

Reply:
Hmm, but then we'll have two different behaviors for errors for the `currentPassword`

<img width="364" alt="Screen Shot 2022-06-09 at 14 44 44" src="https://user-images.githubusercontent.com/16747822/172921421-e79aafc2-db4c-412b-85c1-dd941d454dca.png">
<img width="367" alt="Screen Shot 2022-06-09 at 14 44 12" src="https://user-images.githubusercontent.com/16747822/172921422-97615d3b-1201-447b-9e57-d55e93ee1839.png">

I think it's fairly simple to just always show the errors in place of the `passwordPage.newPasswordPrompt` below the `New Password` text box. That way we can have consistent spacing and placing of error messages and it aligns with what our design team agreed on.